### ISSUE TITLE
termux pip install bug

### ISSUE THREAD

**Issue by petrmachcz** [pymupdf/PyMuPDF#3260]
### Description of the bug

**pip install PyMuPDF**
```
Collecting PyMuPDF                                                                                                                                                                                                                                                                           
  Using cached PyMuPDF-1.23.26.tar.gz (22.0 MB)                                                                                                                                                                                                                                              
  Installing build dependencies: started                                                                                                                                                                                                                                                     
  Installing build dependencies: finished with status 'done'                                                                                                                                                                                                                                 
  Getting requirements to build wheel: started                                                                                                                                                                                                                                               
  Getting requirements to build wheel: finished with status 'done'                                                                                                                                                                                                                           
  Preparing metadata (pyproject.toml): started                                                                                                                                                                                                                                               
  Preparing metadata (pyproject.toml): still running...
...
 (+1m47.5s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build: mkdir -p build/PyMuPDF-aarch64-shared-tesseract-release/source/tools/ ; cc -ffunction-sections -fdata-sections -pipe -O2 -DNDEBUG -DTOFU_CJK_EXT -Iinclude -MMD -MP -o build/PyMuPDF-aarch64-shared-tesseract-release/source/tools/mutrace.o -c source/tools/mutrace.c -fPIC -Wall -Wsign-compare -Ithirdparty/freetype/include -Iscripts/freetype -Ithirdparty/gumbo-parser/src -Ithirdparty/harfbuzz/src -Ithirdparty/libjpeg -Iscripts/libjpeg -Ithirdparty/lcms2/include -DHAVE_LCMS2MT -Ithirdparty/mujs -Ithirdparty/zlib -Ithirdparty/jbig2dec -Ithirdparty/openjpeg/src/lib/openjp2 -DOPJ_STATIC -DOPJ_HAVE_INTTYPES_H -DOPJ_HAVE_STDINT_H -DHAVE_LEPTONICA -Ithirdparty/leptonica/src -DHAVE_TESSERACT -Ithirdparty/tesseract/include -Iscripts/tesseract -Ithirdparty/extract/include -Ithirdparty/extract/src  -DHAVE_PTHREAD                                                                                                                                                                                                                                                 
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build: source/tools/mudraw.c:342:12: error: redefinition of 'kill' as different kind of symbol                                                                                                                 
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:   342 | static int kill = 0;                                                                                                                                                                            
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:       |            ^                                                                                                                                                                                    
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build: /data/data/com.termux/files/usr/include/signal.h:170:5: note: previous definition is here                                                                                                               
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:   170 | int kill(pid_t __pid, int __signal);                                                                                                                                                            
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:       |     ^                                                                                                                                                                                           
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build: source/tools/mudraw.c:601:11: warning: comparison between pointer and integer ('int (*)(pid_t, int)' (aka 'int (*)(int, int)') and 'int') [-Wpointer-integer-compare]                                   
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:   601 |         if (kill == 1)                                                                                                                                                                          
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:       |             ~~~~ ^  ~
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build: source/tools/mudraw.c:608:16: warning: comparison between pointer and integer ('int (*)(pid_t, int)' (aka 'int (*)(int, int)') and 'int') [-Wpointer-integer-compare]                                   
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:   608 |         else if (kill == 2)                                                                                                                                                                      
 (+1m47.6s): -b: m: __main__.py:1562:build: (+0.1s): -b: m: __main__.py:1562:build:       |                  ~~~~ ^  ~
```

### How to reproduce the bug

`pip install PyMuPDF`

### PyMuPDF version

1.23.26

### Operating system

Linux

### Python version

3.11

**Comment by julian-smith-artifex-com** [comment]
The next release of MuPDF works around this problem by renaming the top-level `kill` identifier that's causing this problem. This was originally reported on Android; hopefully the fix will also work for you.

We're expecting the next MuPDF release to come out in the next week or two, and will make a new PyMuPDF release shortly afterwards that uses it.

If you're like you could try building with latest MuPDF master (which has the fix) by running `pip` with `PYMUPDF_SETUP_MUPDF_BUILD="git:--branch master https://github.com/ArtifexSoftware/mupdf.git".

**Comment by julian-smith-artifex-com** [comment]
Should be fixed in current release PyMuPDF 1.24.0. Please test and report here.
